components/style/values/specified/box.rs

```diff
@@ -6,11 +6,15 @@
 
 use Atom;
 use cssparser::Parser;
+#[cfg(feature = "gecko")]
+use gecko_bindings::structs;
 use parser::{Parse, ParserContext};
 use std::fmt;
 use style_traits::{ParseError, ToCss};
 use values::CustomIdent;
 use values::KeyframesName;
+#[cfg(feature = "gecko")]
+use values::computed::box_::{TouchAction as ComputedTouchAction};
```

Comment:
nit: No need for braces for just one import.

Reply:
But this import is redundant since the computed one is just the specified one, so instead of this let's just use `TouchAction` at the relevant places.

Reply:
Ah, yeah, I think it's better to just use `TouchAction` below, thanks!